lib/components/editable_select_component.dart

```diff
@@ -113,6 +113,7 @@ class EditableSelectComponent extends SelectComponent {
   }
 
   get current_input_value => findPart("display_input").value;
+  get name => findPart("input").name;
```

Comment:
Also, as far as I remember, there are multiple inputs in EditableSelectComponent's structure (`<input type="text">` and then `<input type="hidden">` to hold the actual value). I think this code doesn't account for that and simply selects the first input it finds. No?